src/lwc/gsChecklistItem/gsChecklistItem.js

```diff
@@ -0,0 +1,95 @@
+import { LightningElement, api } from 'lwc';
+import { NavigationMixin } from 'lightning/navigation';
+import CumulusStaticResources from '@salesforce/resourceUrl/CumulusStaticResources';
+const gsAssetsImage = CumulusStaticResources + '/gsAssets/step';
+
+/**
+* @description This component render the Sub Section Item info
+*/
+export default class GsChecklistItem extends NavigationMixin(LightningElement) {
+    /**
+    * @description data of the item this component render
+    * @type  GS_AdminSetup.ChecklistItem
+    */
+    @api item = {}
+    /**
+    * @description
+    * @return boolean
+    */
+    get hasSecondaryBtn() {
+        return !!this.item.secondaryBtn;
+    }
+    /**
+    * @description
+    * @return boolean
+    */
+    get hasPrincipalBtn() {
```

Comment:
All references to "principal" need to be changed to "primary"
```suggestion
    get hasPrimaryBtn() {
```